src/style.css

```diff
@@ -2,0 +1,97 @@
+@font-face {
+  font-family: Avenir;
+  src: local('Avenir');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: local('Avenir-Book');
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: local('Avenir-Heavy');
+  font-weight: 900;
+}
+
+.page {
+  font-family: Avenir, serif;
```

Comment:
The font-family declaration includes both 'Avenir' and 'serif'. If 'Avenir' is not available, it will fall back to a generic serif font. Ensure this is the intended behavior.